.github/workflows/coverage.yml

```
@@ -0,0 +1,38 @@
+name: Test Coverage
+
+on:
+  pull_request:
+    branches: [ main, master, develop ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)